rbac/onboarding-secret-generator-role.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: onboarding-secret-generator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - jobs
```

Comment:
why should the onboarding job inturn has perms to get/list/create of job again? Pls note these perms should be on ocs-operator as that's the one creating the job